js/packages/mobile-wallet-adapter-walletlib/android/src/main/java/com/solanamobile/mobilewalletadapterwalletlib/reactnative/MobileWalletAdapterBottomSheetActivity.kt

```diff
@@ -0,0 +1,55 @@
+package com.solanamobile.mobilewalletadapterwalletlib.reactnative
+
+import android.os.Bundle
+import android.view.Gravity
+import android.view.WindowManager
+
+import com.facebook.react.ReactActivity
+import com.facebook.react.ReactActivityDelegate
+import com.facebook.react.ReactRootView
+
+class MobileWalletAdapterBottomSheetActivity : ReactActivity() {
```

Comment:
This is the custom Bottom Sheet activity that we've included in the library and it's styled to support floating window